src/consensus/aft/request.h

```diff
@@ -20,32 +20,44 @@ namespace aft
 
     MSGPACK_DEFINE(rid, caller_cert, raw, frame_format);
 
-    std::vector<uint8_t> serialise()
+    size_t serialised_size() const
     {
-      bool include_caller = false;
-      size_t size = sizeof(rid) + sizeof(include_caller) + sizeof(size_t) +
-        raw.size() + sizeof(enclave::FrameFormat);
+      size_t size = sizeof(rid) + sizeof(bool) + sizeof(size_t) + raw.size() +
```

Comment:
Not for this PR but its been bothering me for a while and I don't think we wrote it anywhere: We should really add a `serialized::size` which does this calculation, since we have a similar pattern in so many places. What we're really saying is "I know I'm going to call `serialized::write(data, size, VECTOR)` later, and I know _that_ is going to write a `size_t` prefix then VECTOR.size bytes", and if we decide to try to pack that a whole bunch of hard-to-grep code is going to break. If we instead used `serialized::size(VECTOR)` as a precursor to `serialized::write`, the prefixing logic could be contained.